pytest_tests/testsuites/services/http_gate/test_http_system_header.py

```diff
@@ -0,0 +1,413 @@
+                logger.info(f"Unable to find {key_to_check}: '{val_to_check}' in {header_att}")
+                return False
+        return True
+
+    @allure.title(
+        f"Validate that only {EXPIRATION_EPOCH_HEADER} exists in header and other headers are abesent"
+    )
+    def validation_for_http_header_attr(self, head_info: dict, expected_epoch: int) -> None:
+        # check that __NEOFS__EXPIRATION_EPOCH attribute has corresponding epoch
+        assert (
+            self.check_key_value_presented_header(
+                head_info, {EXPIRATION_EPOCH_HEADER: str(expected_epoch)}
+            )
+            == True
+        ), f'Expected to find {EXPIRATION_EPOCH_HEADER}: {expected_epoch} in: {head_info["header"]["attributes"]}'
+        # check that {EXPIRATION_EPOCH_HEADER} absents in header output
+        assert (
+            self.check_key_value_presented_header(head_info, {EXPIRATION_DURATION_HEADER: ""})
+            == False
+        ), f"Only {EXPIRATION_EPOCH_HEADER} can be displayed in header attributes"
+        # check that {EXPIRATION_TIMESTAMP_HEADER} absents in header output
+        assert (
+            self.check_key_value_presented_header(head_info, {EXPIRATION_TIMESTAMP_HEADER: ""})
+            == False
+        ), f"Only {EXPIRATION_TIMESTAMP_HEADER} can be displayed in header attributes"
+        # check that {EXPIRATION_EXPIRATION_RFC} absents in header output
+        assert (
+            self.check_key_value_presented_header(head_info, {EXPIRATION_EXPIRATION_RFC: ""})
+            == False
+        ), f"Only {EXPIRATION_EXPIRATION_RFC} can be displayed in header attributes"
+
+    @allure.title(f"Put / get / verify object and return head command result to invoker")
+    def oid_header_info_for_object(self, file_path: str, attributes: dict, create_container: str):
+        oid = upload_via_http_gate_curl(
+            cid=create_container,
+            filepath=file_path,
+            endpoint=self.cluster.default_http_gate_endpoint,
+            headers=attr_into_str_header_curl(attributes),
+        )
+        get_object_and_verify_hashes(
+            oid=oid,
+            file_name=file_path,
+            wallet=self.wallet,
+            cid=create_container,
+            shell=self.shell,
+            nodes=self.cluster.storage_nodes,
+            endpoint=self.cluster.default_http_gate_endpoint,
+        )
+        head = head_object(
+            wallet=self.wallet,
+            cid=create_container,
+            oid=oid,
+            shell=self.shell,
+            endpoint=self.cluster.default_rpc_endpoint,
+        )
+        return oid, head
+
+    @allure.title("[negative] attempt to put object with expired epoch")
+    @pytest.mark.parametrize(
+        "object_size",
+        [pytest.lazy_fixture("simple_object_size")],
+        ids=["simple object"],
+    )
+    def test_unable_put_expired_epoch(self, create_container: str, object_size: int):
```

Comment:
we can use directly simple_object_size as argument in the test if it's only one param

Reply:
done